cypress/features/regression/designSystem/menu.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Design Systems Menu component
+  I want to test the Design Systems Menu component
+    
+  @positive
+  Scenario Outline: Scenario name: Test that all child link elements are part of a scrollable block
+      Given I open "Design System Menu" component page "scrollable-submenu" in no iframe
```

Comment:
1. We can move this step to the `Background:` above the TC;
2. Pass here `scrollable submenu` as a parameter to make it more readable.